client/js/app/README.md

```diff
@@ -0,0 +1,12 @@
+<!-- Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
+
+# Vespa client
+
+This app is work in progress
+
+# Client install and start
+
+    cd src/app
+    nvm install --lts node  # in case your current node.js is too old
+    yarn add vite
+    yarn dev                # then open link, like http://127.0.0.1:5173/
```

Comment:
Isn't it always 3000?

Reply:
it wasn't when I tried ...

Reply:
but it was now, running from the root catalog, not in src/app